pizza-time/src/components/card/card.js

```diff
@@ -136,9 +145,10 @@ class Card extends React.Component {
                   return (
                    
                     <div>
-                    <b>{comment.first_name}</b>{comment.time}<br/>
+                    <b>{comment.first_name}</b>{" "}{comment.time.substring(0,15)}<br/>
```

Comment:
"semantically you would want to declare this as a <p> or a <span> and add the bold or italics as a style.
<span style="font-weight: bold;">...</span>
<span style="font-style: italic;">...</span>

or even just wrapping them in a span or p tag and leaving the  <b> or <i> would work too. 
<span><b></b></span>too. 
<span><i></i></span>"